runtime/js/11_workers.js

```diff
@@ -177,6 +179,13 @@
       const hasSourceCode = false;
       const sourceCode = decoder.decode(new Uint8Array());
 
+      if (specifier.startsWith("./") || type == "classic") {
```

Comment:
`./foo.ts`, `/foo.ts`, and `../foo.ts` should all be relative to `baseUrl`. You should be able to remove this conditional.

Reply:
Ah yeah, I need to catch all three of those. The conditional needs to stay though, the point is to let invalid bare specifiers (unless in classic mode) pass this and hit an error in Rust, rather than being based into a valid absolute specifier.